docs/latest/user/gateway-api-metrics.md

```diff
@@ -0,0 +1,54 @@
+# Gateway API Metrics
+
+Resource metrics for Gateway API objects are available using the [Gateway API State Metrics](https://github.com/Kuadrant/gateway-api-state-metrics) project.
+The project also provides example dashboard for visualising the metrics using Grafana, and example alerts using Prometheus & Alertmanager.
+
+## Prerequisites
+
+Follow the steps from the [Quickstart Guide](quickstart.md) to install Envoy Gateway and the example manifest.
+Before proceeding, you should be able to query the example backend using HTTP.
+
+[Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) is used to install the metrics stack.
+Install Kustomize by following the instructions on the kustomize site.
+Once Kustomize is installed, run the following commands to install the metrics stack, with the Gateway API State Metrics configuration, on your kubernetes cluster.
+
+```shell
+kustomize build github.com/kuadrant/gateway-api-state-metrics.git/examples/kube-prometheus | docker run --rm -i ryane/kfilt -i kind=CustomResourceDefinition | kubectl apply --server-side -f -
```

Comment:
It is possible to release the yaml in upstream, so that people can use it easier

Reply:
Yep, like https://github.com/envoyproxy/gateway/releases/download/latest/install.yaml, this is not blocking this PR, just raise a question.

Reply:
<img width="811" alt="image" src="https://github.com/envoyproxy/gateway/assets/48784001/3c1c9bfc-c371-4935-971f-ec6cecf3afa2">

Reply:
Good idea.
I'll use the kustomize cmds to generate yaml and commit that.
That should avoid potential errors (like in the screenshot above) and remove the dependency on kustomize for installing.

Reply:
@Xunzhuo I've updated the docs to simplify things (no kustomize or docker run needed) by referencing bundle yaml files.
This should also fix the error you were seeing.